proto/kvrpcpb.proto

```diff
@@ -36,8 +36,9 @@ message GetResponse {
     bytes value = 3;
     // True if the key does not exist in the database.
     bool not_found = 4;
-    // Set if ctx.scan_detail = true or meet slow query.
-    ScanDetailV2 scan_detail_v2 = 5;
+    reserved 5;
```

Comment:
This field is not used in a released version. Do we really need to reserve it?

Reply:
Only for our lovely developers who are using the master branch.